docs/platform-specification.md

```diff
@@ -430,6 +430,25 @@ following properties are automatically generated:
 - `{build.variant.path}`: The path to the selected board variant folder (inside the
   [variant platform](#platform-terminology), for example hardware/arduino/avr/variants/micro)
 
+If the platform supports pluggable discovery it may also declare a set of `upload_port.*` properties, these properties
+will be used to identify a board by the discovery process when plugged in.
+
+For example we could declare a series of `upload_port.vid` and `upload_port.pid` for the Uno like so:
+
+    uno.upload_port.vid.0=0x2341
+    uno.upload_port.pid.0=0x0043
+    uno.upload_port.vid.1=0x2341
+    uno.upload_port.pid.1=0x0001
+    uno.upload_port.vid.2=0x2A03
+    uno.upload_port.pid.2=0x0043
+    uno.upload_port.vid.3=0x2341
+    uno.upload_port.pid.3=0x0243
+
+In this case we're using the board's `vid` and `pid` to identify it but `upload_port.*` properties can be anything that
+can help identify a certain board. For more detailed information see the
```

Comment:
```suggestion
In this case we're using the board's USB VID/PID pair to identify it but `upload_port.*` properties can be anything that
can help identify a certain board. For more detailed information see the
```
The platform properties are named with `vid` and `pid`, but the "vid" and "pid" in this sentence are referring to the USB vendor ID (VID) and product ID (PID), which the reader may not be familiar with. This change will at least provide them with some additional search terms. I wanted to linkify the text when I was working on https://arduino.github.io/arduino-cli/dev/platform-specification/#board-vidpid, but I didn't find a straightforward authoritative explanation to link to.